scripts/release_info.py

```diff
@@ -391,7 +493,7 @@ def main():
             '``Thanks to %s, our returning contributors who made this release '
             'possible.``\n' % existing_author_comma_list)
 
-        if parsed_args.personal_access_token:
+        if personal_access_token:
```

Comment:
will this script work without personal_access_token?

Reply:
No.

Reply:
No, we need the token to obtain the repo.